src/test/java/nextstep/optional/study/OptionalMethodUseTest.java

```diff
@@ -0,0 +1,64 @@
+package nextstep.optional.study;
+
+import static org.assertj.core.api.Assertions.*;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.*;
+
+public class OptionalMethodUseTest {
+    Optional<String> opt;
+    String nullName;
+
+    @BeforeEach
+    void setUp() {
+        String name = "optional";
+        opt = Optional.of(name);
+        nullName = null;
+    }
+
+    @Test
+    void orElseTest() {
+        String nullName = null;
+        String name = opt.ofNullable(nullName).orElse("john");
```

Comment:
`Optional#ofNullable(T)`는 정적 메서드입니다.

```suggestion
        String name = Optional.ofNullable(nullName).orElse("john");
```